config/environments/development.rb

```diff
@@ -21,7 +21,7 @@
   # Enable/disable caching. By default caching is disabled.
   if Rails.application.secrets[:redis_url]
     config.action_controller.perform_caching = true
-    config.cache_store = :redis_store
+    config.cache_store = :redis_cache_store, { url: Rails.application.secrets[:redis_url] }
```

Comment:
What does this change do?

Reply:
https://www.engineyard.com/blog/rails-5-2-redis-cache-store

TL;DR Rails comes bundled with their own version of https://github.com/redis-store/redis-rails